discord/flags.py

```diff
@@ -717,12 +717,22 @@ def members(self):
         """
         return 1 << 1
 
-    @flag_value
+    @alias_flag_value
     def bans(self):
-        """:class:`bool`: Whether guild ban related events are enabled.
+        """:class:`bool`: Alias of :attr:`.moderation`.
+
+        .. versionchanged:: 2.4
```

Comment:
Should probably be 2.4.1 (or another version) since 2.4 is already released, we can also consider deprecating it